controller/test/controller_data_packet_ser/controller_data_packet_ser_test.cpp

```diff
@@ -22,7 +22,7 @@ void test_status_serialization() {
                                   PACKET_LEN_MAX, &encoded_data_length);
 
   TEST_ASSERT_EQUAL_INT16(true, status);
-  TEST_ASSERT_EQUAL_INT16(28, encoded_data_length);
+  TEST_ASSERT_EQUAL_INT16(26, encoded_data_length);
```

Comment:
Since this is a fragile test, should we just leave it out?  Or check that the encoded length is greater than some minimal value that we can guarantee is necessary to encode all the data?

Reply:
It varies with nanopb settings and will grow if we keep adding more stuff to status message. Probably just a check it's `>0`?

Reply:
I am fine with simply checking `> 0`.